assets/js/googlesitekit/modules/create-info-store.js

```diff
@@ -79,7 +79,7 @@ export const createInfoStore = ( slug, {
 		 * @return {(string|undefined)} The admin reauthentication URL.
 		 */
 		getAdminReauthURL: createRegistrySelector( ( select ) => ( state, reAuth = true ) => {
-			const { needsReauthentication } = select( CORE_USER ).getAuthentication() || {};
+			const needsReauthentication = select( CORE_USER ).needsReauthentication() || false;
```

Comment:
Seeing this now, it actually needs another small tweak which is more important even than using the selector: If `needsReauthentication` is `undefined`, this shouldn't assume `false` - it should instead return `undefined`, because it means part of the dependency data hasn't loaded yet.